sound/soc/intel/boards/bytcht_cx2072x.c

```diff
@@ -205,9 +205,16 @@ static struct snd_soc_dai_link byt_cht_cx2072x_dais[] = {
 	},
 };
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_BAYTRAIL)
```

Comment:
this is intended, right? If Baytrail is disabled we use "bytcht-cx2072x"?

Reply:
@lyakh yes, I think so. In the else case, the machine driver is used with SST driver and we don't want to change the card name as that is widely used. With SOF, we can still make the change.

Reply:
@kv2019i got it, thanks!

Reply:
what @kv2019i said :-)